WalletWasabi/Crypto/Strobe.cs

```diff
@@ -0,0 +1,318 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace WalletWasabi.Crypto
+{
+	// https://strobe.sourceforge.io/papers/strobe-20170130.pdf
+	// Based on Merlin framework small implementation: https://doc-internal.dalek.rs/src/merlin/strobe.rs.html
```

Comment:
this should also contain a github permalink to https://github.com/dalek-cryptography/merlin/blob/1ed350bbc1d65f0a0697e0c20c48e11ec172c6ff/src/strobe.rs